frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/GitCommandUpdatingCurrentBackgroundable.java

```diff
@@ -54,12 +54,12 @@
 
 @ExtensionMethod(GitMacheteBundle.class)
 @CustomLog
-public abstract class GitCommandUpdatingCurrentBranchBackgroundable extends Task.Backgroundable {
+public abstract class GitCommandUpdatingCurrentBackgroundable extends Task.Backgroundable {
 
   protected final Project project;
   protected final GitRepository gitRepository;
 
-  public GitCommandUpdatingCurrentBranchBackgroundable(
+  public GitCommandUpdatingCurrentBackgroundable(
```

Comment:
Huh I'd consider leaving it here... as kinda exception (or we can assume the rule that `Branch` is only implied in **action class** names... even to be documented in CONTRIBUTING.md?)